projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,7 +18,49 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const getConnectionsDiv = () => {
+	// eslint-disable-next-line no-undef
+	if ( ! connections ) {
+		return null;
+	}
+	// eslint-disable-next-line no-undef
+	const providers = Object.keys( connections );
+	return providers.map( provider => {
+		// eslint-disable-next-line no-undef
+		const connectionIds = Object.keys( connections[ provider ] );
+		const connectionDiv = connectionIds.map( connectionId => {
+			// eslint-disable-next-line no-undef
+			const connection = connections[ provider ][ connectionId ];
+			return (
+				<div>
+					<tr>
+						<th>Name</th>
+						<th>Image</th>
+					</tr>
+					<tr>
+						<td>{ connection.external_display }</td>
+						<td>
+							<img
+								alt="connection_image"
+								src={ connection.profile_picture ? connection.profile_picture : null }
+								height="50px"
+								width="50px"
+							/>
```

Comment:
I'd recommend making the entire `<img>` element conditional, rather than just the `src` attribute. Like this:

`{ connection.profile_picture && <img .... /> }`